source/local/local.go

```diff
@@ -144,17 +152,48 @@ func (s *localSource) GetFeatures() *nfdv1alpha1.Features {
 	return s.features
 }
 
-func parseFeatures(lines [][]byte) map[string]string {
-	features := make(map[string]string)
+func parseComments(line string, fileName string) map[string]interface{} {
+	comments := make(map[string]interface{})
+	if strings.HasPrefix(line, CommentsPrefix) {
+		lineSplit := strings.SplitN(line, "=", 2)
```

Comment:
I think it would be cleaner to drop the `CommentPrefix` at this point. `ExpiryTimeKey` would become just `expiry-time`, without the `# +` prefix

Reply:
How would we differentiate then between comments and features? It can also be ambiguous for users.

Reply:
Comment lines start with a `#`.  This is now already supported in ` master`. We don't need to parse features in `parseComment()`

Reply:
Got it! I was confused by the previous comment above. 😅